aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,27 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## February 14 - March 4, 2022
+
+### Communications
+- **Office Hours** - We have combined our two office hours into one which will be held every Thursday, 10:00 PM IST / 8:00 AM PST. This office hours is open to all. Contact Kamlesh Thakur, if you would like a calendar invite.
+
+### Roadmap Updates 
+- **Page Layouts**
+  - Added support for `xsmall` breakpoint. This is a **breaking change** in the theme and requires instances where users are using small as a breakpoint to shift to mobile designs need to change their check to `xsmall`.
```

Comment:
We haven't merged this yet in the theme so I don't think we should announce this yet. Additionally, we need to create some kind of migration guide to instruct folks how to ensure nothing breaks in their UI. And we should have that created before releasing the theme.

Reply:
that's fair I can take it out. agree we can chat about that since we also need something for the text weak as well